File: javaSE/day_11/src/homework/Demo_02.java
package homework;

import java.util.HashSet;
import java.util.Random;

/*
第二题: 通过代码, 完成如下需求:
        1. 编写一个程序, 获取10个 20~50 之间的随机数.
        //包含20, 50
        2. 要求随机数不能重复, 并将结果打印到控制台上.
        */
public class Demo_02 {
    public static void main(String[] args) {
        //2.创建HashSet集合
        HashSet<Integer> hs = new HashSet<>();

        //3.判断集合长度是否小于等于10,是则添加元素,否则停止添加元素
        while (hs.size() < 10) {
            hs.add(getRandomNumber());
        }
        System.out.println(hs);

    }

    public static int getRandomNumber() {
        //1.编写一个程序, 获取10个 20~50 之间的随机数,包含20, 50
       /* Random random=new Random();
        int number=random.nextInt(31)+20;*/
        int number = new Random().nextInt(31) + 20;
        return number;
    }
}

File: scala/day03/src/cn/itcast/scala/Demo02.scala
package cn.itcast.scala
/*1.2 	在单例对象中定义成员方法
	在object中定义的成员方法类似于Java的静态方法
示例
示例说明
	设计一个单例对象，定义一个能够打印分割线（15个减号）的方法
	在main方法调用该方法，打印分割线
*/
object Demo02 {
  object Print{
  def print()={println("-"*15)}
}

  def main(args: Array[String]): Unit = {
    Print.print()
  }
}
